usr.bin/procstat/procstat.1

```diff
@@ -286,6 +291,58 @@ command
 .It ARGS
 command line arguments (if available)
 .El
+.Ss CHERI information
+Display the process ID, command, and CHERI-specific information:
+.Pp
+.Bl -tag -width "RSTATE" -compact
+.It PID
+process ID
+.It COMM
+command
+.It C
+CHERI support in the process ABI
+.It QUAR
+quarantining status
+.It RSTATE
+in-kernel revoker state
+.It EPOCH
+in-kernel revoker epoch
+.El
+.Pp
+The following ABI CHERI support values may be displayed:
```

Comment:
Should it be "CHERI ABI"?

Reply:
I reworded this, hopefully it's more sensible.